android-test-app/src/main/kotlin/okhttp/android/testapp/TestApplication.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2023 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp.android.testapp
+
+import android.app.Application
+
+class TestApplication: Application() {
```

Comment:
Why was this added? Seems unneeded.

Reply:
Not strictly necessary, but I wanted the full structure. Not sure whether to delete now, might leave and reconsider.